src/extension.ts

```diff
@@ -37,8 +39,13 @@ async function checkRequiredJolieVersion():Promise<void> {
 		const result = stderr.match(/Jolie\s+(\d+\.\d+\.\d+).+/)
 		if (result.length > 1) {
 			const jolieVersion = result[1]
-			if( !semver.satisfies(jolieVersion, versionRequirement) ) {
-				window.showErrorMessage(`This extension requires Jolie version ${versionRequirement}, whereas your version is ${jolieVersion}. Some features may not work correctly. Please consider updating your Jolie installation.`)
+			if( !semver.satisfies(jolieVersion, versionRequirement1) ) {
```

Comment:
Now this just becomes a loop that starts from the end of the list. If we cannot find anything that matches, the error gets out.